Brain_I2C/Brain_I2C_Basic.ino

```diff
@@ -0,0 +1,26 @@
+#include <Wire.h>
+
+void setup() {
+  Wire.begin();        // join i2c bus (address optional for master)
+  Serial.begin(9600);  // start serial for output
+}
+
+void loop() {
+  int sensor_id = 0; //change to desired sensor_id
+  Wire.requestFrom(8, 4); // request 4 bytes from address 8
+  uint8_t index = 0;
+  union floatToBytes {
+      char buffer[4];
+      float voltageReading;
+  } converter;
+  
+  while (Wire.available()){
+    if (sensor_id == 0) { //successive if statements for more IDs
+      converter.buffer[index] = Wire.read();
```

Comment:
So is the idea to have one converter for each sensor? OR use the same converter for multiple different sensors?
I think it makes sense to use the same one, and we can remove the if statement from here if we add a for loop at the top of the loop method that loops through all of our sensors.